tests/components/darksky/test_sensor.py

```diff
@@ -165,11 +165,11 @@ def test_setup_bad_api_key(self, mock_get_forecast):
     @MockDependency('forecastio')
     @patch('forecastio.load_forecast', new=load_forecastMock)
     def test_setup_with_alerts_config(self, mock_forecastio):
-        """Test the platform setup with language configuration."""
+        """Test the platform setup with alert configuration."""
         setup_component(self.hass, 'sensor', VALID_CONFIG_ALERTS)
 
-        state = self.hass.states.get('sensor.dark_sky_summary')
-        assert state is None
+        state = self.hass.states.get('sensor.dark_sky_alerts')
+        assert '0' == state.state
```

Comment:
Please put the expected value last in the assertion.

Reply:
Address in 1324a99